package.json

```diff
@@ -19,8 +19,10 @@
     ]
   },
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=10.0.0",
```

Comment:
Should be >=12.0.0

Reply:
Let's wait for the oclif merge for these?